src/meta/src/storage/mem_meta_store.rs

```diff
@@ -68,10 +68,9 @@ impl Snapshot for MemSnapshot {
 }
 
 impl MemStore {
-    /// Get a global shared in-memory store.
-    pub fn shared() -> Self {
-        static STORE: LazyLock<MemStore> = LazyLock::new(MemStore::default);
-        STORE.clone()
+    /// Create a new in-memory store.
+    pub fn new() -> Self {
```

Comment:
As we're using simulated etcd in the recovery test, I've made this instance not-shared.

Reply:
see it 👌